### DISCUSSION TITLE
关于1panel使用代理后不能访问网站的问题

### DISCUSSION THREAD

**Discussion by MillerFour** [1Panel-dev/1Panel#5545] [Q&A]
1panel的开发者你们好，首先感谢你们的开源心血， 我是1个月前开始使用1panel这个运维管理工具，期间遇到几个问题如下：

1. 我有一台在硅谷的服务器，当我部署好相关网站后，

- 当我打开代理，则不能访问这些网站
- 当我关闭代理，则能访问

  因为我日常中工作中经常需要开启代理服务，所以请问这个问题该如何解决？

2. 某天突然服务器完全登录不了，包括但不限于：ssh、mysql（navicat远程访问）、redis（redis client 远程访问）等，后验证发现，凡是苏州的IP均无法访问，但外地的朋友均能访问（无锡、黄冈、深圳等）

以上两个问题均对我产生了极大的困扰，麻烦开发者看到后能帮助我排查下，感谢（抱拳~）

**Comment by zhengkunwang223** [comment]
我们在 WAF 模块有限制地区访问网站的功能，但是是专业版功能
其他地方没有限制 你这个情况 大概率还是你机器的问题

**Reply by MillerFour** [reply]
好的，谢谢回复
